Resources/Public/Css/recordlist.css

```
@@ -46,19 +46,36 @@ main.recordlist ul.pagination input.form-control {
     display: flex;
     flex-wrap: wrap;
     gap: 6px;
 }
 
+.badge-hidden {
+    display: none !important;
+}
+
+.badge-show-more,
+.badge-show-less {
+    border: 1px dashed currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)